vault/logical_system_integ_test.go

```diff
@@ -220,6 +273,28 @@ func testSystemBackendMock(t *testing.T, numMounts int, backendType logical.Back
 	return cluster
 }
 
+func ensureCoresSealed(t *testing.T, c *vault.TestCluster) {
```

Comment:
Can we maybe not have this function copied around somewhere yet again?

Let's start migrating such utility functions to a helper/testutil or so package.

Reply:
Moved to testing.go since its a `TestCluster` operation